cosmwasm/processor.ts

```diff
@@ -85,23 +75,28 @@ export async function mainProcessor(processorFn: ProcessorFn, options: Options,
 
     const client = await prepareClient(options.mnemonic, configManager.axelar.rpc, GasPrice.fromString(configManager.axelar.gasPrice));
 
-    await processorFn(client, configManager, options, args, fee);
+    const processedOptions: Options = {
+        ...options,
+        deposit,
+    };
+
+    await processorFn(client, configManager, processedOptions, args, fee);
```

Comment:
hmm, why do we even bother processing the deposit in advance, in a generic processor that's used by all scripts? We don't even know if governance will be used at this stage vs EOA or if the processor is doing something completely different. Ideally, such logic is executed just-in-time when it's needed (in this case within `executeByGovernance`). It should have all the `options` to compute it at that time. For comparison, stellar/EVM scripts don't do such use case specific processing in the generic processor